simulator/Field.py

```diff
@@ -1,21 +1,58 @@
-#!/usr/bin/env python3
+
 """
 File:          Field.py
 Author:        Moungsung Im 
-Last Modified: Binit on 9/23
+Last Modified: Moungsung on 10/8
 """
 
 import Buttons
```

Comment:
This was an error I made when I initialized the Field.py skeleton class, it should be `from simulator import Buttons`